source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -261,15 +263,22 @@ FilterConfig::FilterConfig(const ExternalProcessor& config,
                           config.response_attributes()),
       immediate_mutation_checker_(context.regexEngine()),
       thread_local_stream_manager_slot_(context.threadLocal().allocateSlot()) {
-  // Validate processing mode configuration.
-  verifyProcessingModeConfig(config);
+
+  // Validate processing mode configuration
+  auto status = verifyProcessingModeConfig(config);
+  if (!status.ok()) {
```

Comment:
I think the ultimate goal here is to avoid the exception throw. This PR doesn't achieve as it will eventually the exception will be thrown here. It will be a larger refactor in ext_proc

Here is relevant discussion. https://github.com/envoyproxy/envoy/pull/34942#discussion_r1825852719

Reply:
happy to see this as an intermediate step

Reply:
I agree this PR is one step closer to final goal, as it merges multiple exception-throws to one place.

Reply:
Agreed this PR made some progress.  However, we can do better than this as we are specially targeting this area now.